Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:14.11.0-buster-slim
```

Comment:
Just curious. Any reason to use a specific version here and on the mongo container?

Since this is used on a dev environment, I think it would be fine to use `latest` on both of them.

What do you think? Sorry for not mentioning on the first review 😅

Reply:
Note: I tried it using `latest` on both and it also worked perfectly.

Reply:
> Any reason to use a specific version here and on the mongo container?

I was considering the possibility of using Dockerfile in the production environment in the future, but if it is only for development, latest is good. 👍  (Any plans to do so?)

Reply:
In node image, I seem to be good with `slim` tag because `latest` is about six times larger.
https://hub.docker.com/_/node?tab=tags

Reply:
Slim seems like a good option!

The website is hosted on AWS S3 right now, and the database on mongo Atlas. I'm satisfied with that for the moment. If the app evolved into having an API (which will probably happen one day), docker will definitely seem like a good option for deployment.

Thanks for your work @tsubasaogawa!